src/js/angular/core/directives/queryeditor/templates/query-editor.html

```diff
@@ -211,7 +211,7 @@ <h6 class="alert alert-warning no-icon additional" ng-show="!queryIsRunning && !
                     <h6>Error<span ng-show="yasr.results.getException().status > 0"> {{yasr.results.getException().status}}: {{yasr.results.getException().statusText}}</span></h6>
                 </div>
 
-                <div class="plaintext">{{yasr.results.getException().responseText}}</div>
+                <div class="plaintext" style="white-space: pre-wrap">{{yasr.results.getException().responseText}}</div>
```

Comment:
What about globally declaring a class, e.g. `pre-wrap-white-space` ?
Inline styles often lead to issues (boiler plate, refactoring, overriding)